qiskit-superstaq/qiskit_superstaq/superstaq_backend.py

```diff
@@ -262,13 +262,15 @@ def ibmq_compile(
         self,
         circuits: Union[qiskit.QuantumCircuit, Sequence[qiskit.QuantumCircuit]],
         dynamical_decoupling: bool = True,
+        dd_strategy: str = "dynamic",
```

Comment:
Curious: what strings are supported for `dd_strategy`? Is it just like "static" or "dynamic", and if so, maybe the docstring can indicate to the user those allowed strategies?

Reply:
Great question! Our key settings for `dd_strategy` are `dynamic`, and `static_context_aware`. We are planning to include a notebook that describes the differences between these, but maybe it would be helpful to include a small description within the docstring as well!

Reply:
I think a notebook explanation sounds great and ideal! Otherwise, depending on preference, maybe the docstring can be appended with something along the lines: 

```
 dd_strategy: Type of method used for placing dynamical decoupling operations; either "dynamic" (default setting) or "static_context_aware".
```

Reply:
I went ahead and added a bit more of a complete definition for all the settings - what do you think? Then we'll dive into the details in the example notebook!

Reply:
It's more than sufficient and quite helpful to know!

Reply:
Thanks so much!!